jython/DispatcherSystem/MoveTrain.py

```diff
@@ -589,32 +593,72 @@ def loadTrainFromTrainInfo2(self, info, overRideType, overRideValue, overRideMax
             log.warn("failed to create Active Train '{}'", info.getTrainName())
             return -1
         return 0
-
+            
     ## ***********************************************************************************
-
+    
     ## sound routines
-
-    ## ***********************************************************************************
-
-
-    # use external "nircmd" command to "speak" some text  (I prefer this voice to eSpeak)
-    def speak(self,msg) :
-        #if self.logLevel > 1: print("about to speak",msg)
-        #java.lang.Runtime.getRuntime().exec('Z:\\ConfigProfiles\\jython\\sound2\\nircmd speak text "' + msg +'"')
-        my_dir = jmri.util.FileUtil.getExternalFilename('program:jython/DispatcherSystem/programs')
-#         if self.logLevel > 1: print "nircmd" + my_dir+'/nircmd'
-#         java.lang.Runtime.getRuntime().exec(my_dir+'/nircmd speak text "' + msg +'"')
-        return
-
-    def announce(self, fromblockname, toblockname, speak_on, direction, instruction):
+    
+    ## ***********************************************************************************            
+        
+    def getOperatingSystem(self):
```

Comment:
I would prefer if you used the OS selection that's built into JMRI itself:

https://www.jmri.org/JavaDoc/doc/jmri/util/SystemType.html

But that's not a requirement.